sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -209,6 +213,7 @@ public class EventHubClientBuilder implements
      */
     public EventHubClientBuilder() {
         transport = AmqpTransportType.AMQP;
+        identifier = UUID.randomUUID().toString();
```

Comment:
It's possible people hold onto the same builder and build multiple clients from it. Initialising it here will always have the same identifier.